test/test_models.py

```diff
@@ -31,12 +31,51 @@ def test_models(self):
             m = models[k]
             passed = False
             for i in xrange(5):
-                content = m.analyze(html)
+                content = m.analyze(self._html)
                 if actual_content[k].encode('utf-8') == content:
                     passed = True
                     break
             self.assertTrue(passed)
 
+    def test_content_and_content_comments_extractor(self):
+        content = content_extractor.analyze(self._html)
+        content_comments = content_comments_extractor.analyze(self._html)
+
+        passed = False
+        for i in xrange(5):
+            actual_content, actual_content_comments = \
+                content_and_content_comments_extractor.analyze(self._html)
+            passed = actual_content == content and (
+                actual_content_comments == content_comments)
+            if passed:
```

Comment:
I thinks it's better to assert the individual equality conditions because a) it'll be easier to diagnose failures, and b) it'll run faster if it fails.